### DISCUSSION TITLE
Remapping buttons

### DISCUSSION THREAD

**Discussion by SkateLifeAway** [darthcloud/BlueRetro#503] [Q&A]
Don't seem to know how to properly remap buttons so I can use switch controller on GameCube using blue retro. B button is acting like I'm pressing a. Help

**Comment by darthcloud** [comment]
This is normal, BlueRetro buttons default mapping is based on their position not their name, switch B is best approximation for GC A location.

If you want to remap check the documentation in the wiki:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#25---example-buttons-mapping

**Comment by SkateLifeAway** [comment]
Thanks, i will check the link out. Hopefully I can get it to remap.

**Comment by SkateLifeAway** [comment]
Finally was able to figure out how to change button layout to use switch
controller how I like but now that I have installed the blue retro, games
are lagging on my picoboot running Swiss.

**Reply by darthcloud** [reply]
Which controller brand you use?

**Comment by SkateLifeAway** [comment]
Nintendo Pro Controller
